app/src/tests/appConfig.test.ts

```diff
@@ -0,0 +1,29 @@
+import config from "../config/appConfig";
+
+describe("api service", () => {
+
+    const OLD_ENV = process.env;
+
+    beforeEach(() => {
+        jest.resetModules(); // Important - it clears the cache
+        process.env = { ...OLD_ENV }; // Make a copy
+    });
+
+    afterAll(() => {
+        process.env = OLD_ENV; // Restore old environment
+    });
+
+    test("uses default config by default", () => {
+        expect(config.apiUrl()).toBe("http://localhost:8080");
+    });
+
+    test("uses production config if node_env is production", () => {
+        /* eslint-disable */
```

Comment:
Could this be a more specific eslint rule disable for this line, or does it set off too many alarm bells?

Reply:
Think it's just the one but also how much do we care about linting one line in a test :woman_shrugging: